aws-lambda-layer/README.md

```diff
@@ -0,0 +1,44 @@
+# FreeTDS Lambda Layer
+
+Build and publish FreeTDS as an AWS Lambda layer.
+
+In order to use the TinyTDS gem with ruby AWS Lambda Functions, we need to have access to FreeTDS binaries that AWS does not provide with its execution environment.
+
+The solution is to use AWS Lambda Layers.
+
+The provided deploy script builds FreeTDS on the same architecture that runs in AWS Lambda and, using the AWS CLI, publishes the artifact as an AWS Lambda layer for functions to use.
+
+> A Lambda deployment package is a .zip file that contains your code and any dependencies. The Lambda execution environment is based on a specific Amazon Linux AMI and kernel version. Any native binaries that are used in a Lambda deployment package must be compiled in this environment, and only 64-bit binaries are supported.
+> 
+> https://aws.amazon.com/premiumsupport/knowledge-center/lambda-linux-binary-package/
+
+## Requirements
+
+- Ruby 2.5.3
+- Docker
+- AWS CLI
+
+## Usage
+
+- Run `rake deploy`
```

Comment:
Should there be an example for what to do when all you want is to work on a lambda function locally?

Reply:
No, I don't think so.  If you want to work on a lambda function locally you technically don't need this layer.  It's only important when deploying to AWS.

When you want to build a function locally, we make use of a container that has the binaries built into it already which we have called "veracross/api-build-environment" which highjacks the SAM CLI build step.